lib/column/ipv4.go

```diff
@@ -66,39 +71,93 @@ func (col *IPv4) ScanRow(dest interface{}, row int) error {
 	return nil
 }
 
+// appendIPv4Str appends bytes of the IPv4-formatted string to result byte array.
+// If IP is not valid V4 error will be returned.
+func appendIPv4Str(data []byte, strIp string) ([]byte, error) {
+	ip := net.ParseIP(strIp)
+	if ip == nil {
+		return nil, &ColumnConverterError{
+			Op:   "Append",
+			To:   "IPv4",
+			Hint: "invalid IP format",
+		}
+	}
+	return appendIPv4(data, ip)
+}
+
+// appendIPv4 appends bytes of IPv4 to result byte array.
+// If IP is not valid V4 error will be returned.
+func appendIPv4(data []byte, ip net.IP) ([]byte, error) {
+	ip = ip.To4()
+	if ip == nil {
+		return nil, &ColumnConverterError{
+			Op:   "Append",
+			To:   "IPv4",
+			From: "IPv6",
+			Hint: "invalid IP version",
+		}
+	}
+	return append(data, IPv4ToBytes(ip)...), nil
+}
+
+func (col *IPv4) appendIPv4(ip net.IP) (err error) {
```

Comment:
Do we need these one liner methods?  Not sure they really refactor a huge amount and just make the code harder to follow

Reply:
Removed